src/Slider.jsx

```diff
@@ -29,87 +30,110 @@ function killEvent(ev) {
   ev.preventDefault();
 }
 
-class Button extends React.Component {
-  render() {
-    return <button {...this.props} type="button" />;
-  }
-}
-
 const propTypes = {
+  ...withStylesPropTypes,
+
+  // Automatically adds a top position for large when enabled
+  autoAdjustVerticalPosition: PropTypes.bool,
+
   // the algorithm to use
   algorithm: PropTypes.shape({
     getValue: PropTypes.func,
     getPosition: PropTypes.func,
   }),
+
   // any children you pass in
   children: PropTypes.node,
-  // standard class name you'd like to apply to the root element
+
+  // class name prop
   className: PropTypes.string,
```

Comment:
Is this used?

Reply:
className props should be avoided anyways, if possible